src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -1546,6 +1594,62 @@ protected override void HandleThreadsRequestAsync(IRequestResponder<ThreadsArgum
             responder.SetResponse(response);
         }
 
+        public static ProtocolMessages.Module ConvertToModule(in MODULE_INFO debugModuleInfo, IDebugModule2 module)
+        {
+            var mod = new ProtocolMessages.Module(RegisterDebugModule(module), debugModuleInfo.m_bstrName)
```

Comment:
This will be registering the module for both `load` and `unload`.
You only want to register for `load`.